index.d.ts

```diff
@@ -1330,7 +1408,7 @@ declare namespace dashjs {
 
         setTextTrack(idx: number): void;
 
-        getBitrateInfoListFor(type: MediaType): BitrateInfo[];
+        getBitrateInfoListFor(type: MediaType): QualityInfo[];
```

Comment:
Why was this type changed?

Reply:
In general I am not sure that the definition of "BitrateInfo" is correct...you defined it as a class but use it as an array object.
I changed it for "getBitrateInfoListFor" as I have seen during my testing it has more properties compared to "BitrateInfo" and they are the same like "QualityInfo".